src/components/PropsTables.astro

```diff
@@ -25,14 +27,27 @@ const propsData = await getPropsData(propComponents)
 ---
 
 {
-  propsData
-    .filter((comp: any) => !!comp)
-    .map((component: any) => (
-      <PropsTable
-        key={component.name}
-        componentName={component.name}
-        componentDescription={component.description}
-        componentProps={component.props}
-      />
-    ))
+  <Stack hasGutter>
+    <StackItem>
+      <AutoLinkHeader
+        headingLevel="h2"
+        className="pf-v6-c-content--h2"
+        id="props-table"
+      >
+        Props
+      </AutoLinkHeader>
+    </StackItem>
+    {propsData
+      .filter((comp: any) => !!comp)
+      .map((component: any) => (
+        <StackItem>
```

Comment:
You'll need to move the `key` from the `PropsTable` component up to this `StackItem` now that it's the top level component being returned by the map.